app/_src/gateway/production/monitoring/readiness-check.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Readiness Check
+content_type: tutorial
+---
+
+This tutorial will guide you through the process of using the Node Readiness Endpoint, which provides a reliable and simple way to determine if Kong is ready to serve user requests.
+
+The readiness check endpoint returns a `200 OK` response when Kong is ready, or a `503 Service Temporarily Unavailable` response when it's not. This is useful for load balancers and other tools that need to monitor the readiness of Kong instances.
+
```

Comment:
```suggestion
The status endpoint responds back with a 'message' with the reason for unreadiness. This can be very helpful to debug situations where the user expects that the node should be ready but is not.

```

https://www.merriam-webster.com/dictionary/unready says unreadiness is a word.